public/apps/configuration/constants.tsx

```diff
@@ -265,37 +279,23 @@ export const INDEX_PERMISSIONS: string[] = [
   'indices:admin/shards/search_shards',
   'indices:admin/shrink',
   'indices:admin/synced_flush',
-  'indices:admin/template/delete',
-  'indices:admin/template/get',
-  'indices:admin/template/put',
   'indices:admin/types/exists',
   'indices:admin/upgrade',
   'indices:admin/validate/query',
   'indices:data/read/explain',
   'indices:data/read/field_caps',
   'indices:data/read/field_caps*',
   'indices:data/read/get',
-  'indices:data/read/mget',
-  'indices:data/read/mget*',
-  'indices:data/read/msearch',
-  'indices:data/read/msearch/template',
-  'indices:data/read/mtv',
-  'indices:data/read/mtv*',
   'indices:data/read/point_in_time/create',
   'indices:data/read/point_in_time/delete',
   'indices:data/read/point_in_time/readall',
-  'indices:data/read/scroll',
-  'indices:data/read/scroll/clear',
   'indices:data/read/search',
   'indices:data/read/search*',
   'indices:data/read/search/template',
   'indices:data/read/tv',
-  'indices:data/write/bulk',
-  'indices:data/write/bulk*',
```

Comment:
backend says equal to indices:data/write/bulk - should it include the * as well?